ext-apiserver/pkg/apis/cluster/v1alpha1/machineset_types_test.go

```diff
@@ -0,0 +1,79 @@
+
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+
+package v1alpha1_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	. "k8s.io/kube-deploy/ext-apiserver/pkg/apis/cluster/v1alpha1"
+	. "k8s.io/kube-deploy/ext-apiserver/pkg/client/clientset_generated/clientset/typed/cluster/v1alpha1"
+)
+
+var _ = Describe("MachineSet", func() {
```

Comment:
@krousey was working on removing ginkgo from other parts of the repo; do we really want to add more here?

Reply:
The test is now passing. This is also auto-generated code. For whatever refactoring @krousey is doing, we will apply the same here. Until here, I'd to keep the consistency between different resource type.